forte/mrdsrg-spin-integrated/master_mrdsrg.h

```diff
@@ -38,6 +38,9 @@ class MASTER_DSRG : public DynamicCorrelationSolver {
     /// Compute energy
     virtual double compute_energy() = 0;
 
+    /// Compute the DSRG gradient
+    virtual psi::SharedMatrix compute_gradient() = 0;
```

Comment:
This function may not be pure virtual. We could throw a runtime error saying it is only implemented for DSRG-MRPT2.

Reply:
FIXED